gulpfile.js

```diff
@@ -85,6 +86,10 @@ function makeTryMorphir() {
     return make('cli', 'src/Morphir/Web/TryMorphir.elm', 'web/try-morphir.html')
 }
 
+async function addLitComponents() {
+    return src(['./cli/web/insight.js', './cli/web/morphir-insight-element.js']).pipe(concat('insight.js')).pipe(dest('./cli/web/'))
```

Comment:
What is concat doing here?

Reply:
The above line means: take this list of sources, concat them into a file named insight.js, and write it to this destination. It effectively takes insight.js and append the contents of morphir-insight-element.js